src/Components/Text/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import './index.css';
+
+export default class Text extends Component {
+  render(){
+    return(
+      <p className="mainText"></p>
```

Comment:
Same thing as with the `Subtitle` component, we need a way to pass text into this component, can you add a prop?